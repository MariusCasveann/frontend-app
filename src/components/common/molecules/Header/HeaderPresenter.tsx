import * as React from 'react';
import { Icon } from 'backoffice-shared-components';
import { UserMenu } from './../../../authentication';
import { Layout } from 'antd';
import './Header.css';

export default () => (
    <Layout.Header>
        <div className="header-menu">
            <Icon type="grid" className="grid-icon" />
            <UserMenu />
        </div>
    </Layout.Header>
);
