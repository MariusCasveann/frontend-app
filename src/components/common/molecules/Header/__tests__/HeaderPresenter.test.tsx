import * as React from 'react';
import renderer from 'react-test-renderer';
jest.mock('backoffice-shared-components', () => ({ Icon: 'Icon' }));
jest.mock('./../../../../authentication', () => ({ UserMenu: 'UserMenu' }));
import HeaderPresenter from './../HeaderPresenter';

describe('Render header', () => {
    it('renders correctly', () => {
        const tree = renderer.create(<HeaderPresenter />).toJSON();
        expect(tree).toMatchSnapshot();
    });
});
