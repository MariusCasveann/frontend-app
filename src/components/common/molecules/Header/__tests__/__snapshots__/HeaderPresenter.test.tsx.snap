// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render header renders correctly 1`] = `
<header
  className="ant-layout-header"
>
  <div
    className="header-menu"
  >
    <Icon
      className="grid-icon"
      type="grid"
    />
    <UserMenu />
  </div>
</header>
`;
